operations/operations/company-agreements.md

```diff
@@ -4,44 +4,38 @@
-
-CAOs by area include:
-
-* Natalie Jew \(HR\) - Employment contracts, benefit contracts, vendor contracts (HR)
-* Aneal Vallurupalli \(Finance\) - Sales contracts, corporate contracts, vendor contracts
-* Tim Quock \(Accounting and Procurement\) - Sales contracts, corporate contracts, vendor contracts
-
 ## Who can send contracts for signature?
 
-The following staff are approved to send contracts to be initialed and e-signed:
+The following staff are approved to send contracts to be e-signed:
 
+## Approved Mattermost Staff
 * Lynn Conway - Primary for staff agreements and consulting agreements
 * Amy Nicol - Primary for vendor contracts and corporate agreements
 * Jeff Dynda - Primary for sales agreements and customer supplier forms
 * Natalie Jew
 * Aneal Vallurupalli
 * Tim Quock
+* Jamie Hurewitz
 
 ## What are e-sign completion expectations?
 
 * E-sign request sent on weekdays before 4pm Palo Alto time should be signed by 9pm Palo Alto on the same day and if sent on a weekend signed by 9pm Palo Alto time on the following Monday provided one of the following is true:
-  * **For e-sign from Mattermost staff member:** The e-sign is sent by a Mattermost staff member and either prepared by or initialed by an approved [Contract Accountability Owner, or CAO](company-agreements.md#who-are-contract-accountability-owners-caos). E-signs sent from approved Mattermost staff are preferred since they are archived automatically in our e-sign system.
-  * **For e-sign from outside the company:** The e-sign is prepared by a non-Mattermost staff member \(potentially a vendor, partner or customer\) and approved by a [CAO](company-agreements.md#who-are-contract-accountability-owners-caos) via email to Ian Tien and Amy Nicol ideally ahead of receiving the e-sign request.
-    * Within 5 business days of the e-sign completion the [CAO](company-agreements.md#who-are-contract-accountability-owners-caos) or designee should send Amy Nicol a link to the Box.com URL containing the fully executed signature to confirm the archiving of the contract is complete, since archiving is not automatic.
+  * **For e-sign from Mattermost staff member:** The e-sign is sent by a Mattermost staff member and must have Legal approval. E-signs sent from Approved Mattermost staff are preferred since they are archived automatically in our e-sign system.
+  * **For e-sign from outside the company:** The e-sign is prepared by a non-Mattermost staff member \(potentially a vendor, partner or customer\) and Legal approval is provided via email to Ian Tien and Amy Nicol ideally ahead of receiving the e-sign request.
+    * Within 5 business days of the e-sign completion the contract owner or Mattermost staff submitting for signature should send Amy Nicol a link to the Box.com URL containing the fully executed signature to confirm the archiving of the contract is complete, since archiving is not automatic.
```

Comment:
```suggestion
    * Within 5 business days of the e-sign completion the contract owner or Mattermost staff member submitting for signature should send Amy Nicol a link to the Box.com URL containing the fully executed signature to confirm the archiving of the contract is complete, since archiving is not automatic.
```